core/src/main/java/hudson/FilePath.java

```diff
@@ -2266,12 +2267,15 @@ private Integer writeToTar(File baseDir, DirScanner scanner, OutputStream out) t
 
     /**
      * Reads from a tar stream and stores obtained files to the base dir.
+     * @since TODO: the method uses 
```

Comment:
+1 for keeping `@since` Since is valid javadoc and defines from what version appeared